bin/make_test

```diff
@@ -13,6 +13,7 @@ tempdir=$(mktemp -d)
 slug="test-project"
 cookiecutter . --directory "$1" --output-dir "$tempdir" --no-input slug="$slug" "${@:2}"
 cd "$tempdir"/"$slug"
+if [ -z "${CI+x}" ]; then make services; fi
```

Comment:
This is the cookiecutter's own tests. Same as on CI, the tests now need to run `make services` to start the DB. This runs `make services` only if the envvar `CI` is not set, because CI uses GitHub Actions's services thing, we don't need to start the services in Docker Compose on CI.